contracts/btc-light-client/src/bitcoin.rs

```diff
@@ -193,9 +214,70 @@ fn check_block_header_context(
         });
     }
 
+    // BIP 113
+    let height = prev_block_height + 1;
+
+    let csv_height = match chain_params.network {
+        Network::Bitcoin => 419328, // 000000000000000004a1b34462cb8aeebd5799177f7a29cf28f2d1961716b5b5
+        Network::Testnet => 770112, // 00000000025e930139bac5c6c31a403776da130831ab85be56578f3fa75369bb
+        Network::Signet => 1,       // Always active unless overridden
+        Network::Regtest => 1,      // Always active unless overridden
+        _ => unreachable!("Unsupported network"),
+    };
```

Comment:
These height checks are not part of `CheckBlockHeaderContext`, why do we add them here?

Reply:
Thanks for pointing this out! I got confused by BIP 113, which defines the transaction locktime constraint using the median time past. The CSV height check is only required when validating that constraint, which we don't need in the light client, so it's unnecessary in this context.